packages/good-design/src/core/web3/Converter.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { memo, useCallback, useState } from "react";
+import { Box, Divider, Input, Text, VStack } from "native-base";
+
+import { CentreBox } from "../layout/CentreBox";
+
+const ConverterCircle = () => (
+  <svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
why are we not using svg files?

Reply:
We could, but we use both 'defined as react-component' and 'imported from svg-file'
one can be used as <SvgComponent /> the other is used as src for Image

Reply:
importing svg files should be the default choice.